firebase-firestore/src/androidTest/java/com/google/firebase/firestore/FirestoreTest.java

```diff
@@ -570,6 +570,10 @@ public void testQueriesAreValidatedOnClient() {
     query.whereArrayContainsAny("y", Arrays.asList("cat"));
     // In on different field works.
     query.whereIn("y", Arrays.asList("cat"));
+    // Array contains any with varargs works.
+    collection.whereArrayContainsAny("a", "b", "c", 42);
```

Comment:
Could you add a test that verifies `collection.whereArrayContainsAny("field", null)` actually works (not just compiles)?